webapp/packages/core-blocks/src/LabeledIconButton.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2021 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import type { ButtonHTMLAttributes } from 'react';
+import styled, { css } from 'reshadow';
+
+import { composes, useStyles } from '@cloudbeaver/core-theming';
+import type { RequireAtLeastOne } from '@cloudbeaver/core-utils';
+
+import { Button } from './Button';
+import { IconOrImage } from './IconOrImage';
+
+const styles = composes(
+  css`
+    Button {
+      composes: theme-ripple from global;
+    }
+  `,
+  css`
+    Button {
+      height: 100%;
```

Comment:
not sure about this style, is we actually want to stretch height up to the container?